indico/web/client/styles/partials/_icons.scss

```diff
@@ -151,7 +152,8 @@ i[class^='icon-']::before {
 .icon-checkbox-unchecked::before {
   content: '\e433';
 }
-.icon-checkmark::before {
+.icon-checkmark::before,
+%icon-checkmark {
```

Comment:
@foxbunny ^

Reply:
Ok, let me see what I can do.

Reply:
Btw, I was intending to duplicate all of those as placeholders. I can probably come up with a way to automate that so that it's not (as) painful to update.

Reply:
Ok, there we go.

https://codepen.io/hayavuk/pen/KKbGzbW

I'll include this in the instructions.

Reply:
@ThiefMaster Please try it before I push the changes, let me know if it works for you.

Reply:
What it does is:

- Changes all selectors to use dashes instead of underscores
- Lower-cases all selectors
- Changes double to single quotes
- Adds the placeholders

Reply:
I'm noticing that the icomoon JSON you have in the repo contains some icons that aren't in the _icons.scss. Just an FYI.

Reply:
The extra ones are just alias names that are added by default. The unfortunately need to be removed manually. Alternatively, maybe only the first class name for any given `content` value could be kept since that's already the one we're using anyway.

Your script to convert it works fine. I'd suggest to add it in `bin/maintenance/` though, to avoid relying on some external codepen.

It could actually work directly on the _icons.scss and modify the parts between `/* Icomoon's icon definitions */` and `/* End of icomoon's icon definitions */` (so you'd just paste the icomoon css there and then run that script to fix things)

Reply:
Actually, even better than pasting into that area would be the script taking the icomoon css file as input (or having the user paste it into the script's stdin). In either case I would change the begin/end markers to something where it's clearer that it's meant for a machine and not a human.

`/* BEGIN GENERATED ICOMOON CSS */` and `/* END GENERATED ICOMOON CSS */` for example

Reply:
I think that might be a separate PR. If we take this further, we're also relying on icomoon as an external service to convert the icon. Ideally you'd just generate the icon font locally from a folder containing the icon files and do that as part of the webpack build.

Reply:
Basically, I suggest we just move with the codepen for now and optimize the workflow later.

Reply:
You mean like duplicate the existing declaration specifically for the placeholder?

Reply:
The reason I put it there is because `::before` selectors cannot be extended, btw.

Reply:
Pushed some changes. Let me know if that's what you meant.

Reply:
yes, exactly :)